File: Question2/Primes.cs
using System;

namespace SimcorpInterview
{
    public static class Primes {
        /// Write an algorithm that returns the first <n> prime numbers (простi числа)
        /// Assume n <= 100
        private static int[] FirstPrimes(int numberOfPrimes)
        { 
            return new int[] {2, 3, 5};
        }

        public static void Main(string[] args)
        {
            foreach (var n in FirstPrimes(3))
            {
                Console.WriteLine(n);
            }
        }
    }
}

File: Question2F/Primes.fs
module SimcorpInterview.FSharp.Primes

/// Write an algorithm that returns the first <n> prime numbers (простi числа)
/// Assume n <= 100
let firstPrimes (n : int) : int array =
    [|2; 3; 5|]

firstPrimes 3 |> Array.iter (printfn "%i")
